src/imxrt1010/flexio.rs

```diff
@@ -0,0 +1,37 @@
+//! FlexIO pin configurations
+
+use super::pads::{gpio::*, gpio_ad::*, gpio_sd::*};
+use crate::{flexio::Pin, Daisy};
+
+//
+// FLEXIO1
+//
+flexio!(module: 1, offset: 0, pad: GPIO_08, alt: 4, daisy: None);
+flexio!(module: 1, offset: 1, pad: GPIO_09, alt: 4, daisy: None);
+flexio!(module: 1, offset: 2, pad: GPIO_10, alt: 4, daisy: None);
+flexio!(module: 1, offset: 3, pad: GPIO_11, alt: 4, daisy: None);
+flexio!(module: 1, offset: 4, pad: GPIO_12, alt: 4, daisy: None);
+flexio!(module: 1, offset: 5, pad: GPIO_13, alt: 4, daisy: None);
+flexio!(module: 1, offset: 6, pad: GPIO_SD_00, alt: 4, daisy: None);
+flexio!(module: 1, offset: 7, pad: GPIO_SD_01, alt: 4, daisy: None);
+flexio!(module: 1, offset: 8, pad: GPIO_SD_02, alt: 4, daisy: None);
+flexio!(module: 1, offset: 9, pad: GPIO_SD_03, alt: 4, daisy: None);
+flexio!(module: 1, offset: 10, pad: GPIO_SD_04, alt: 4, daisy: None);
+flexio!(module: 1, offset: 11, pad: GPIO_SD_05, alt: 4, daisy: None);
+flexio!(module: 1, offset: 12, pad: GPIO_SD_06, alt: 4, daisy: None);
+flexio!(module: 1, offset: 13, pad: GPIO_SD_07, alt: 4, daisy: None);
+flexio!(module: 1, offset: 14, pad: GPIO_SD_08, alt: 4, daisy: None);
+flexio!(module: 1, offset: 15, pad: GPIO_SD_09, alt: 4, daisy: None);
+flexio!(module: 1, offset: 16, pad: GPIO_SD_10, alt: 4, daisy: None);
+flexio!(module: 1, offset: 17, pad: GPIO_SD_11, alt: 4, daisy: None);
+flexio!(module: 1, offset: 18, pad: GPIO_SD_12, alt: 4, daisy: None);
+flexio!(module: 1, offset: 19, pad: GPIO_SD_13, alt: 4, daisy: None);
+flexio!(module: 1, offset: 20, pad: GPIO_AD_00, alt: 4, daisy: None);
+flexio!(module: 1, offset: 21, pad: GPIO_AD_09, alt: 4, daisy: None);
+flexio!(module: 1, offset: 22, pad: GPIO_AD_10, alt: 4, daisy: None);
+flexio!(module: 1, offset: 23, pad: GPIO_AD_11, alt: 4, daisy: None);
+flexio!(module: 1, offset: 24, pad: GPIO_AD_12, alt: 4, daisy: None);
+flexio!(module: 1, offset: 25, pad: GPIO_AD_13, alt: 4, daisy: None);
+flexio!(module: 1, offset: 26, pad: GPIO_AD_14, alt: 4, daisy: None);
```

Comment:
imxrt-ral defines the FlexIO instance with a `0` const generic. These `module: 1` definitions will cause a type mismatch for anyone using type constraints to check module equality.

The recommendation is to use a 0. The 1010 FlexPWM pin implementations take this approach, and it lets us type check against the imxrt-ral 1010 FlexPWM instance.

https://github.com/imxrt-rs/imxrt-iomuxc/blob/03c3f2baac5dd6f09df36d82bffe2d7d954d71a9/src/imxrt1010/flexpwm.rs#L1-L4

Mind if I change this 1 to a 0?